tests/fuzz/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "typst-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+arbitrary = { version = "1.3.1", features = ["derive"] }
+libfuzzer-sys = "0.4"
+typst-syntax = { path = "../../crates/typst-syntax" }
+typst-library = { path = "../../crates/typst-library" }
+typst-render = { path = "../../crates/typst-render" }
+typst = { path = "../../crates/typst" }
+comemo = "0.3"
+unscanny = "0.1"
```

Comment:
there are some unused dependencies in here. could you also move this to the workspace-style dependencies used everywhere else?

Reply:
Good catch, I removed unscanny, everything else is used, and now inherited from the workspace.

Reply:
where is `arbitrary` used?

Reply:
The trait `arbitrary::Arbitrary` is used in the `fuzz_target!()` macro. i.e. the `text: &str` can be replaced with any type the implements arbitrary. But good call, I might be able to remove it from the Cargo.toml.

Reply:
removed